actionview/test/template/url_helper_test.rb

```diff
@@ -541,13 +541,13 @@ def test_mail_with_options
 
   def test_mail_to_with_img
     assert_dom_equal %{<a href="mailto:feedback@example.com"><img src="/feedback.png" /></a>},
-      mail_to('feedback@example.com', '<img src="/feedback.png" />'.html_safe)
+      mail_to('feedback@example.com', raw('<img src="/feedback.png" />'))
   end
 
   def test_mail_to_with_html_safe_string
     assert_dom_equal(
       %{<a href="mailto:david@loudthinking.com">david@loudthinking.com</a>},
-      mail_to("david@loudthinking.com".html_safe)
+      mail_to("david@loudthinking.com")
```

Comment:
Updated. I had removed where tests passed without html_safe. Looks like this verification needs it.